src/components/Search.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import TextField from "@material-ui/core/TextField";
+import {searchTodo} from '../actions/actions' ;
+import PropTypes from "prop-types";
+
+function Search(props){
+    return (
+      <div id="searchField">
+        <i className="fas fa-search"></i>
```

Comment:
Component/element with no children can use self-closing bracket.

Reply:
Got it